recipes/python-libarchive-c/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+export DETERMINISTIC_BUILD=1
```

Comment:
Absolutely nothing... as it depends on some nix-specific patches to Python. Removed.